src/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -52,16 +49,19 @@ interface State {
   openSuggestions: boolean
 }
 
+const SuggestionsPopoverContainer = styled(Box)`
+  position: absolute;
+  z-index: 1000;
```

Comment:
Mind experimenting with the lowest possible index?

Reply:
Hah... Sure. I'm not even sure why the contact button was above this pop up. So I went with the random really high number. Will definitely try to make it a more reasonable number.